extensions/2.0/Khronos/KHR_audio_emitter/README.md

```diff
@@ -0,0 +1,371 @@
+
+#### `bufferView`
+
+The index of the bufferView that contains the audio data. Use this instead of the audio source's uri property.
+
+#### `mimeType`
+
+The audio's MIME type. Required if `bufferView` is defined. Unless specified by another extension, the only supported mimeType is `audio/mpeg`.
+
+#### `uri`
+
+The uri of the audio file. Relative paths are relative to the .gltf file.
+
+### Audio Sources
+
+Audio sources reference audio data and define playback properties for it. Audio sources may be used by zero to many audio emitters.
+
+#### Property Summary
+
+|              | Type      | Description                                                                                               | Default value        |
+| ------------ | --------- | --------------------------------------------------------------------------------------------------------- | -------------------- |
+| **gain**     | `number`  | Unitless linear multiplier against original audio file volume used for determining audio source loudness. | 1.0                  |
+| **loop**     | `boolean` | Whether or not to loop the specified audio when finished.                                                 | false                |
+| **autoPlay** | `boolean` | Whether or not to play the specified audio when the glTF is loaded.                                       | false                |
+| **audio**    | `number`  | The index of the audio data assigned to this clip.                                                        | Required, no default |
+
+#### `gain`
+
+Unitless linear multiplier against original audio file volume used for determining audio source loudness. If not specified, the audio source volume gain is `1.0`.
+
+This value is linear, a value of `0.0` is no volume, `0.5` is half volume, `1.0` is the original volume, `2.0` is double the volume, etc. The final volume of the audio is a combination of this value, the audio emitter's gain, and if the audio emitter is positional, the relative positions of the emitter and listener.
+
+#### `loop`
+
+Whether or not to loop the specified audio when finished. If `false` or not specified, the audio source does not loop.
+
+#### `autoPlay`
+
+Whether or not to play the specified audio when the glTF is loaded. If `false` or not specified, the audio source does not play automatically.
+
+#### `audio`
+
+The index of the audio data in the "audio" array assigned to this audio source. This property is required.
+
+### Audio Emitter
+
+Audio emitters define how audio sources are played back. Emitter properties are defined at the document level and are references by nodes. Audio may be played globally or positionally. Positional audio has further properties that define how audio volume scales with distance and angle.
```

Comment:
Should a node's scale affect volume?

Reply:
It would make sense to me if the global scale of the node affected the refDistance and range / maxDistance, but not the volume. This way if you scale up a glTF scene with audio, the audio can be heard at the same volume in the same places relative to that scene, and that scene's audio can be heard from farther away.

Reply:
I struggle to know how to implement this in a way that content creators can anticipate. It's very common that I will attach some component to an item and scale it to fit it into a space. Very common also that I'll make something in blender and forget to apply those scales. Making audio volume change based on that scale seems high risk for people to have unintended loud or quiet audio. 

I do like the idea outside of this spec and would implement something like a `KHR_audio_behavior` or `KHR_audio_affector` that can define these behaviors and maybe be a solid place to reference things like effect chains. Would be cool to use things like the node structure to do scale->volume->reverb(more if bigger, less if smaller)->distortion(when super big)

Reply:
Understood! I don't have a preference, and I think explicitly leaving this behavior undefined could be OK. The question has come up with `KHR_lights_punctual`, particularly in AR/VR/XR applications, so I just wanted to anticipate it if possible here.

Reply:
Randomly stumbling over this, from some GitHub notification, completely without context: 
**Iff** a node `scale` should affect an audio volume, then _non-uniform_ scaling will raise questions that should be addressed clearly and explicitly early in the process.